docs/advanced/dcrtime.md

```diff
@@ -0,0 +1,105 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Servers.svg" /> Dcrtime
+
+---
+
+Dcrtime is a timestamping application that allows users to store hashes of arbitrary data on the Decred blockchain. Inspired by Peter Todd's [opentimestamps](https://petertodd.org/2016/opentimestamps-announcement), dcrtime was originally built to store timestamped hashes of data from [Politeia](../../governance/politeia/overview), Decred's proposal system, on the Decred blockchain. 
+
+Although dcrtime was created for Politeia, it is expected to have generic utility as a timestamping service. Dcrtime is implemented as a client and server application, with all code open source and  available in the [dcrtime repo](https://github.com/decred/dcrtime). Decred also offers a free timestamping service through its public mainnet server. We expect this to be particularly useful in scenarios where transparency, accountability and time-ordering are of key importance, in either a public or private context (e.g. computer security, data integrity and various compliance contexts).
+
+In this page, we list some [notable uses](#notable-uses) of dcrtime, as well as describe the high-level [timestamping process](#timestamping-process), [dcrtime architecture](#dcrtime-architecture), and [dcrtime's implementation details](#dcrtime-implementation) via examples.
+
+## Notable uses
+
+Below are some notable uses of dcrtime:
+
+* **Politeia:** all data submitted to Politeia (proposals, votes, comments, comment upvotes/downvotes, etc.) is commited to a public [git repository](https://github.com/decred-proposals/mainnet) every hour. Each commit is then hashed and timestamped onto the Decred blockchain using dcrtime. 
+* **Dcrtimestamptweet:** A Twitter bot ([@dcrtimestampbot](https://twitter.com/dcrtimestampbot), [git repo](https://github.com/tiagoalvesdulce/dcrtimestamptweet)) that uses dcrtime to timestamp a thread when a chosen keyword is mentioned. This bot uses [IPFS](https://en.wikipedia.org/wiki/InterPlanetary_File_System) as a decentralized database to store the tweet thread. 
+* **Timestamping election data:** In the 2018 Brazillian presidential election, candidate Fernando Haddad [used dcrtime](https://cointelegraph.com/news/brazilian-presidential-candidate-uses-blockchain-to-publish-government-plan) to timestamp his government plan and information about his campaign. 
+
+## Timestamping process
+
+Dcrtime allows a nearly unlimited number of hashes to be timestamped. It does this by creating a single merkle root from any number of hashes and including it in a transaction on the Decred blockchain. This process can be summarized as follows:
+
+1. A 32-byte hash is created for each document or piece of data to be timestamped.
```

Comment:
Use `1.` for all items in a numbered list please

Reply:
Ah, keep forgetting. Done